src/Manifolds.jl

```diff
@@ -435,68 +435,119 @@ function shortest_geodesic(M::Manifold, x, y, T::AbstractVector)
 end
 
 """
-    vector_transport_to!(M::Manifold, vto, x, v, y)
+    [Ξ,κ] = tangent_orthonormal_basis(M,x,v)
+
+returns both an orthonormal basis `Ξ` for the tangent space of `x` on the
+[`Manifold`](@ref) `M`, which as its first direction contains a normalized
+tangent vector of `v` as well as the sectional curvatures `κ` required for
+computing Jacobi fields, i.e. the first value (along `v`) is zero.
+"""
+tangent_orthonormal_basis(M::Manifold,x,v) = error("A tangent orthonormal basis not implemented on $(typeof(M)) at $(typeof(x)) with first direction $(typeof(v)).")
```

Comment:
`Ξ` should be an `AbstractVector` of tangent vectors, right? Might be worth adding to the docstring.

Reply:
Yes of course. Thanks for the tipp.

Reply:
Yes, exactly that, one can even state the length of that vector (it's `manifold_dimension(M)` long)